src/classes/TDTM_Config_API.cls

```diff
@@ -96,4 +96,17 @@ global class TDTM_Config_API {
         TDTM_ProcessControl.toggleTriggerState('npe01__OppPayment__c', 'CRLP_Rollup_TDTM', false);
     }
 
+
+    /*******************************************************************************************************
+    * @description Method to disable all TDTM triggers for the current code execution context
+    * @return void
+    ********************************************************************************************************/
+    global static void disableAllTriggers() {
```

Comment:
I would leave this as a public method so that we're not locked into forever. We've exposed the ability to call the method already through our new callable api, so it's not necessary to have another entry point as a global method.